### ISSUE TITLE
Easier assignment of initial Entity data

### ISSUE THREAD

**Issue by codediodeio** [redis/redis-om-node#1]
Currently, you must assign each entity property manually:
   
``` 
    let album = repository.createEntity()
    album.artist = "Mushroomhead"
```

However, it would be nice to create this initial data with an existing object (like a request body). One potential solution is an optional arg to the `createEntity` method:

``` 
    let data = { artist:  "Mushroomhead" }
    let album = repository.createEntity(data)
```

**Comment by guyroyse** [comment]
I like this idea a lot. I'll get it into the backlog.

**Comment by guyroyse** [comment]
I'll start working on this next week and will have it out before Christmas Eve.

**Comment by guyroyse** [comment]
I pushed this to NPM a couple of hours ago. It is included in version 0.1.5.
